### ISSUE TITLE
how to modify the leraning rate

### ISSUE THREAD

**Issue by bruce1408** [open-mmlab/mmcv#2304]
Interrupt the model training, resume the model from previous checkpoint, the learning rate is still the last saved value, how to modify the learning rate manual？

config：
```
optimizer = dict(type='AdamW', lr=2e-3, weight_decay=0.01)
optimizer_config = dict(grad_clip=None)
lr_config = dict(
    policy='step',
    warmup='linear',
    warmup_iters=500,
    warmup_ratio=0.001,
    step=[16, 22, 40, 45])
```

**Comment by zhouzaida** [comment]
Hi, if you do not want to resume the optimizer state, you could try to load from the checkpoint rather than resuming.
